api/mapping/views.py

```diff
@@ -1195,6 +1208,259 @@ def get_context_data(self, **kwargs):
             context = {"user_string": query.user_string, "results": json_response}
             return context
 
+def run_nlp(request):
+
```

Comment:
Any chance you can wrap it around a function in nlp_services?

Reply:
I was actually going to do this when I was still working on the "old" version on the NLP code. Do you want this to happen for this PR? Or shall I do it once we've merged and I've branched off master again?